.github/workflows/build.yml

```diff
@@ -61,6 +61,37 @@ jobs:
       - name: Lint 
         run: |
           script/lint
+          
+  style:
+    # ubuntu 18.04 comes with lein + java8 installed
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          submodules: 'true'
+          
+      - name: Style 
+        run: |
+          script/style
+                 
+  carve:
+    # ubuntu 18.04 comes with lein + java8 installed
```

Comment:
I'm using `java11`. Should we try to stick to one version?

Reply:
It's should not matter for most Clojure feature (time API's are in new jdks'). 

I haven't reinstalled java for a long time. Still running Java8. I'm not sure if I'm unique, but I wouldn't enforce it on the dev part, unless it becomes a problem.

On CI we might want to test against multiple version indeed. Or choose a version that we specifically support. I guess java11 makes sense, although I'm still focussed on Java8 myself. I don't see big issues here